sdk/textanalytics/Azure.AI.TextAnalytics.Legacy.Shared/src/GeneratorStubs/InternalTypes.cs

```diff
@@ -32,7 +32,7 @@ internal partial class KeyPhrasesTaskParameters { }
     internal partial class SentimentAnalysisTaskParameters { }
 
     [CodeGenModel("PiiCategory")]
-    internal readonly partial struct PiiEntityCategory { }
+    internal readonly partial struct PiiEntityLegacyCategory { }
```

Comment:
This was needed because there's a `PiiEntityCategory` in the `Azure.AI.TextAnalytics` namespace.  Because the legacy generated code does not fully qualify the type name, the compiler was matching the wrong class in the main project.